src/states/massachusetts/main.py

```diff
@@ -7,8 +7,9 @@
 
 re_city = re.compile(r'^(.*?)\n')
 re_addr = re.compile(r'\n(.*?)\n\s*Phone', re.MULTILINE | re.DOTALL)
-re_phone = re.compile(r'\n\s*Phone:\s*(.*?)\n')
-re_fax = re.compile(r'\n\s*Fax:\s*(.*?)\n')
+
```

Comment:
I'm afraid that we may mis some slightly malformed numbers.  Could we be more generous on the intake and then write a more robust substitution in `re_to_e164`?